lamps-rfc7030-csrattrs.mkd

```diff
@@ -417,22 +427,33 @@ The Base64:
 
 ### ASN.1 DUMP output
 
-1. The challengePassword attribute is included to indicate that the CSR should include this value.
+The CsrAttrs structure contains:
 
-2. An ecPublicKey attribute is provided with the value secp384r1 to indicate what kind of key should be submitted.
+1. The challengePassword attribute is included to indicate that the
+   CSR should include this value.
 
-3. An extensionRequest container with an OID 1.3.6.1.1.1.1.22 (macAddress), but without a value, to indicate that the CSR should include an X.509v3 extension with this value.
+2. An ecPublicKey OID is provided with the value secp384r1 to
+   indicate what kind of public key should be submitted.
 
-4. The ecdsaWithSHA384 OID is included to indicate what kind of hash is expected to be used for the self-signature of the PKCS#10 CSR structure.
+3. An extensionRequest attribute with a requirement to include the
+   subjectDirectoryAttributes extension.  The macAddress attribute
+   is required in that extension, with the OID 1.3.6.1.1.1.1.22,
+   but without a value.  This indicates that the CSR should include
+   a subjectDirectoryAttributes extension, and the value for this
+   extension is is required to include the macAddress.
+
+4. The ecdsaWithSHA384 OID is included to indicate what kind of hash
+   is expected to be used for the self-signature in the PKCS#10 CSR.
 
 ~~~~
 {::include examples/rfc7030-example01.csrattr.dump}
 ~~~~
 
 ## EST server requires a specific subjectAltName extension
 
-This example is the same as the previous one except that instead of the OID
-for a macAddress, a subjectAltName is specified as the only Extension element.
+This example is the same as the previous one except that instead of
+the OID for a macAddress, a subjectAltName is specified as the only
+Extension element.
```

Comment:
Since this example is not as much the same as the previous one, point out that here a concrete value is required to be included:
```suggestion
This example is similar to the previous one but instead of requiring to fill in a macAddress, it requires including a subjectAltName with a given email address: "potato@example.com".
```